backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -1326,6 +1326,13 @@ def make_content_creation(self, data: typing.Dict[str, typing.Any]) -> object:
         return ContentCreation(**data)
 
 
+class ContentMinimalSchema(marshmallow.Schema):
```

Comment:
ContentDigestSchema contains many data, i'm not sure we do want to return so much data just to get breadcrumb.

On the other side, this mean one more schema for content (we have 3 with this on)

Reply:
That's ok, we should really review the usage of digest schemas because this is quite strange that almost all the object is returned in it…